client/state/data-layer/wpcom/posts/revisions/index.js

```diff
@@ -31,10 +33,15 @@ export function normalizeRevision( revision ) {
 	}
 
 	return {
-		...revision,
+		...omit( revision, [ 'title', 'content', 'excerpt', 'date', 'date_gmt', 'modified', 'modified_gmt' ] ),
 		...flow(
 			pick( [ 'title', 'content', 'excerpt' ] ),
 			mapValues( ( val = {} ) => val.rendered )
+		)( revision ),
+		...flow(
+			pick( [ 'date_gmt', 'modified_gmt' ] ),
```

Comment:
first guess would be: `_.pick` is the non-fp version? which means it returns an empty object `{}`, and `flow` doesn't like non-functions as an arg?